packages/app/playwright/e2e/ChangePassword.test.ts

```diff
@@ -8,7 +8,7 @@ test.describe('ChangePassword', () => {
   let browser: Browser;
   let page: Page;
 
-  test.beforeAll(async () => {
+  test.beforeEach(async () => {
```

Comment:
Why the change for `beforeEach` instead of `beforeAll`?

Reply:
I changed this because during the last iteration of tests, I need the error cache to be cleared after the second test.

I'm reverting, since we are no longer logging these kind of errors.